pantalaimon/daemon.py

```diff
@@ -849,18 +897,27 @@ async def send_message(self, request):
         if request.match_info["event_type"] == "m.reaction":
             encrypt = False
 
+        msgtype = request.match_info["event_type"]
+
         # The room isn't encrypted just forward the message.
         if not encrypt:
+            content = ""
+            try:
+                content = await request.json()
+            except (JSONDecodeError, ContentTypeError):
+                return self._not_json
+
+            content_msgtype = content["msgtype"]
+            if content_msgtype in ["m.image", "m.video", "m.audio", "m.file"]:
+                try:
+                    content = await self._map_media_upload(content, request, client)
```

Comment:
here I should change `request.content` with `content` but I don't know  how to do it. :)

Reply:
done!